src/hooks/useActionCenterSubscriber/useLimitOrderActionSubscriber.tsx

```diff
@@ -0,0 +1,394 @@
+import { ExternalLinkIcon } from '@chakra-ui/icons'
+import {
+  Box,
+  Flex,
+  Icon,
+  Link,
+  Text as CText,
+  useColorModeValue,
+  usePrevious,
+  useToast,
+} from '@chakra-ui/react'
+import { OrderStatus } from '@shapeshiftoss/types'
+import { bnOrZero, fromBaseUnit } from '@shapeshiftoss/utils'
+import { useEffect, useMemo } from 'react'
+import { TbCircleCheckFilled, TbCircleXFilled } from 'react-icons/tb'
+import { useTranslate } from 'react-polyglot'
+import { v4 as uuidv4 } from 'uuid'
+
+import { useLocaleFormatter } from '../useLocaleFormatter/useLocaleFormatter'
+
+import { useLimitOrdersQuery } from '@/components/MultiHopTrade/components/LimitOrder/hooks/useLimitOrders'
+import { actionSlice } from '@/state/slices/actionSlice/actionSlice'
+import { ActionStatus, ActionType, isLimitOrderAction } from '@/state/slices/actionSlice/types'
+import { PriceDirection } from '@/state/slices/limitOrderInputSlice/constants'
+import {
+  selectBuyAmountCryptoBaseUnit,
+  selectInputSellAmountCryptoBaseUnit,
+} from '@/state/slices/limitOrderInputSlice/selectors'
+import { LimitOrderSubmissionState } from '@/state/slices/limitOrderSlice/constants'
+import { limitOrderSlice } from '@/state/slices/limitOrderSlice/limitOrderSlice'
+import {
+  selectActiveQuoteBuyAsset,
+  selectActiveQuoteExpirationTimestamp,
+  selectActiveQuoteId,
+  selectActiveQuoteLimitPrice,
+  selectActiveQuoteSellAsset,
+  selectLimitOrderSubmissionMetadata,
+} from '@/state/slices/limitOrderSlice/selectors'
+import {
+  selectLimitOrderActionByCowSwapQuoteId,
+  selectLimitOrderActionByLimitOrderId,
+  selectOpenLimitOrderActionsFilteredByWallet,
+} from '@/state/slices/selectors'
+import { store, useAppDispatch, useAppSelector, useSelectorWithArgs } from '@/state/store'
+
+type UseLimitOrderActionSubscriberProps = {
+  onDrawerOpen: () => void
+}
+
+export const useLimitOrderActionSubscriber = ({
+  onDrawerOpen,
+}: UseLimitOrderActionSubscriberProps) => {
+  const dispatch = useAppDispatch()
+  const translate = useTranslate()
+  const toastColor = useColorModeValue('white', 'gray.900')
+
+  const {
+    number: { toCrypto },
+  } = useLocaleFormatter()
+
+  const activeQuote = useAppSelector(limitOrderSlice.selectors.selectActiveQuote)
+  const sellAsset = useAppSelector(selectActiveQuoteSellAsset)
+  const buyAsset = useAppSelector(selectActiveQuoteBuyAsset)
+  const sellAmountCryptoBaseUnit = useAppSelector(selectInputSellAmountCryptoBaseUnit)
+  const buyAmountCryptoBaseUnit = useAppSelector(selectBuyAmountCryptoBaseUnit)
+  const quoteId = useAppSelector(selectActiveQuoteId)
+  const { currentData: ordersResponse } = useLimitOrdersQuery()
+  const toast = useToast({
```

Comment:
Make sure to feature flag me as well as spot toasts 🙏🏽

Reply:
Yeah finally it's not needed at all as pollers are inside ActionCenter which is hidden if the featureflag is off!